pkgs/tools/video/gopro/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, ffmpeg
+, imagemagick
+, makeWrapper
+, mplayer
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gopro";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "KonradIT";
+    repo = "gopro-linux";
+    rev = version;
+    sha256 = "0sb9vpiadrq8g4ag828h8mvq01fg0306j0wjwkxdmwfqync1128l";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    chmod +x gopro
+    cp gopro $out/bin
```

Comment:
`install -Dm755 gopro -t $out/bin` should do the same as all of the above three lines.

Reply:
oh, nice, I like it.

Reply:
And it's included in `coreutils`, so almost always there :)